engine/class_modules/sc_death_knight.cpp

```diff
@@ -1261,6 +1261,7 @@ struct death_knight_t : public player_t {
     std::vector<timespan_t> amz_use_time;
     bool amz_specified = false;
     double average_cs_travel_time = 0.4;
+    timespan_t first_ams_cast = timespan_t::from_seconds(20);
```

Comment:
could probably just be `timespan_t first_ams_cast = 20_s;`

Reply:
Is there a practical difference or just style?